app/forum/templates/forum/base.html

```diff
@@ -51,4 +51,36 @@
         <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
     </body>
+
+    <footer>
+        <script src="{% static 'js/theme.js' %}"></script>
+
+        <script>
+            function loadThemeCSS(theme){
+                var fileref = document.createElement("link");
+                fileref.rel = "stylesheet";
+                fileref.type = "text/css";
+                fileref.href = "{% static 'css/light-theme.css' %}";
+
+                if(theme == "dark"){
+                    fileref.href = "{% static 'css/dark-theme.css' %}"
+                }else if(theme == "cyber"){
+                    fileref.href = "{% static 'css/cyber-theme.css' %}"
+                }
+
+                document.getElementsByTagName("head")[0].appendChild(fileref);
+            }
+        </script>
+        
+        <div id="theme-wrapper">
+            <label>Theme:</label>
+            <select id="theme" onchange="onThemeChange();">
```

Comment:
What if we use [switch like this](https://codepen.io/mburnette/pen/LxNxNg) instead of select? We can have 2 themes - light and dark. I think this will fit the minimal design I am aiming for.

What do you think about it?

Reply:
That would definitely work better when using only 2 themes.
Also, nice switch you got there, that one is great.

Should we keep this in the footer too?